examples/create_letters_from_csv/letter.py

```diff
@@ -0,0 +1,149 @@
+# usage:
+#   python letter.py input.csv
+#   logs to csv's in the output directory
+
+import os
+import sys
+import lob
+import csv
+import datetime
+
```

Comment:
`sys.path.insert(0, os.path.abspath(__file__+'../../..'))`
This allows you to still run the examples using the local version of the wrapper if you still haven't installed the wrapper through pip.